CRM/ManualDirectDebit/Hook/Alter/ContactDetailReport.php

```diff
@@ -0,0 +1,157 @@
+<?php
+use CRM_ManualDirectDebit_Batch_BatchHandler as BatchHandler;
+
+/**
+ * Class CRM_ManualDirectDebit_Hook_Alter_ContactDetailReport.
+ */
+class CRM_ManualDirectDebit_Hook_Alter_ContactDetailReport {
+
+  /**
+   * Batches
+   *
+   * @var array
+   */
+  protected $_batches = [];
+
+  /**
+   * Handle the hook.
+   *
+   * @param string $varType
+   * @param CRM_Report_Form_Contact_Detail|array $var
+   * @param CRM_Report_Form_Contact_Detail $reportForm
+   */
+  public function handle($varType, &$var, &$reportForm) {
```

Comment:
in this class and in CRM/ManualDirectDebit/Hook/Alter/ContributeDetailReport.php

can you change all the other methods to be private and just keep this as public